core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -11,25 +11,75 @@
 @ConfigRoot
 public class PackageConfig {
 
+    /**
+     * @deprecated Use {@link Type#JAR} instead
+     */
+    @Deprecated
     public static final String JAR = "jar";
```

Comment:
```suggestion
    public static final String JAR = BuiltInType.JAR.name();
```

Reply:
And similarly the rest.

Reply:
Heh, yeah.  Duh.  :smile:

Reply:
Made the suggested changes, though used `BuiltInType.JAR.name` instead because the `name()` method would return the enum name "JAR" rather than the expected value "jar".